packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -18,7 +18,7 @@ import {
   findPropertyValue,
   tryFindPropertyValueAsBoolean,
 } from '../commands/config/configUtils';
-import * as nameHelpers from '../utils/nameHelpers';
+import * as nameHelpers from '@react-native-windows/telemetry/src/utils/nameHelpers';
```

Comment:
See the above comment for properly exporting the functions you need and consuming them here.